examples/ovault-evm/deploy/MyOvault.ts

```diff
@@ -41,17 +41,21 @@ const deploy: DeployFunction = async (hre) => {
         })
         deployedContracts.assetOFT = assetOFT.address
         console.log(
-            `Deployed contract: ${DEPLOYMENT_CONFIG.asset.contract}, network: ${hre.network.name}, address: ${assetOFT.address}`
+            `Deployed contract: ${DEPLOYMENT_CONFIG.AssetOFT.contract}, network: ${hre.network.name}, address: ${assetOFT.address}`
+        )
+    } else if (DEPLOYMENT_CONFIG.vault.assetOFTAddress) {
```

Comment:
can just use else here right? just cuz an `else if` would make me think that there is another default case on an `else` and there is no other `else` 
```suggestion
    } else {
```

Reply:
Let's keep it `else if` since it makes it clear what the reason is for this happening.

If someone extends the deploy script for whatever reason they may have alternate flows and this just makes it clear what this flow specifically is for.